.changeset/cold-guests-train.md

```diff
@@ -0,0 +1,11 @@
+---
+"@comet/cms-admin": minor
+"@comet/cms-site": minor
+"@comet/cms-api": minor
+---
+
+Authenticate site preview without AuthProxy
```

Comment:
This is true for our setup, but a more general title would be: "Create site preview JWT in the API"